doc/mgr/dashboard.rst

```diff
@@ -999,7 +999,7 @@ and has read-only access to other scopes.
 
 1. *Create the user*::
 
-   $ ceph dashboard ac-user-create bob mypassword
+   $ ceph dashboard ac-user-create bob -i <file-containing-password>
```

Comment:
I think you missed the RGW commands:
```
set-rgw-api-access-key
set-rgw-api-secret-key
```
And while reviewing docs, I see another candidate for the `-i <>` replacement:
```
ceph dashboard iscsi-gateway-add <scheme>://<username>:<password>@<host>[:port]
```

Reply:
> I think you missed the RGW commands:
> 
> ```
> set-rgw-api-access-key
> set-rgw-api-secret-key
> ```
@epuertat yeah, the documentation work is not completed as it's still a draft.

Regarding 
> 
> And while reviewing docs, I see another candidate for the `-i <>` replacement:
> 
> ```
> ceph dashboard iscsi-gateway-add <scheme>://<username>:<password>@<host>[:port]
> ```
Good catch.

Reply:
Updated documentation and ```ceph dashboard iscsi-gateway-add``` command secured.